src/main/java/frc/io/subsystems/IO.java

```diff
@@ -14,25 +17,37 @@ public static IO getInstance() {
 
     private IO() {
```

Comment:
If you have time please change this class to use the same methodology as TeleopControl ie.

```
    public void runCycle() {
		for (TeleopComponent t : this.components) {
			t.calculate();
		}
	}

	public void disable() {
		for (TeleopComponent t : this.components) {
			t.disable();
		}
	}

	public void initialize() {
		for (TeleopComponent t : this.components) {
			t.firstCycle();
		}
	}
```

Reply:
Will look into that!